tests/acceptance/features/apiGraph/getUser.feature

```diff
@@ -1,4 +1,4 @@
-@api @skipOnOcV10
+@api @skipOnOcV10 @skipOnStable2.0
```

Comment:
I think the other scenarios for this feature runs on `stable-2.0`. My be you should put it on your newly added scenrios which donot run on `stable-2.0`

Reply:
yes, you are right